test/verify/check-packagekit

```diff
@@ -54,9 +54,15 @@ class TestUpdates(MachineCase):
         m.start_cockpit()
         b.login_and_go("/updates")
 
+        # PK starts from a blank state, thus should force refresh
+        b.wait_in_text("#state", "Refreshing")
+        b.wait_present("#app div.progress-bar")
+
         # no repositories at all, thus no updates
         b.wait_present(".content-header-extra td button")
         b.wait_in_text("#state", "No updates pending")
+        b.wait_present(".content-header-extra td.text-right span")
+        self.assertEqual(b.text(".content-header-extra td.text-right span"), "Last checked: a few seconds ago")
```

Comment:
OK, I can restrict it to just do `wait_in_text("Last checked:")`, which is sufficient to reproduce the former bug about not showing this at all.